briskedu/brisk-learning
# Add ability to set service through URL

Summary:
Allows us to set service

Test Plan:
1. Confirm we can override to sapling or gptzero
2. Confirm not parameter defaults to fallback
